docs/en/connector-v2/source/Kudu.md

```diff
@@ -110,8 +110,8 @@ sink {
 env {
   # You can set engine configuration here
   execution.parallelism = 1
-  job.mode = "STREAMING"
-  execution.checkpoint.interval = 5000
+  job.mode = "BATCH"
+  checkpoint.interval = 5000
```

Comment:
Since pr https://github.com/apache/seatunnel/pull/5914 batch job will not config `checkpoint.interval` default. So I think you need rebase dev and remove `checkpoint.interval` config in batch job.